airflow/configuration.py

```diff
@@ -41,6 +41,22 @@ def expand_env_var(env_var):
         else:
             env_var = interpolated
 
+def run_bash_command(command):
```

Comment:
when i move this, it creates a circular dependency between configuration.py and utils.py